packages/web-react/src/components/Dropdown/README.md

```diff
@@ -16,17 +16,18 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
-| `renderTrigger`    | `(render: DropdownRenderProps) => ReactNode` | -             | no       | Properties for trigger render                                   |
-| `enableAutoClose`  | `boolean`                                    | `true`        | no       | Enables close on click outside of Dropdown                      |
-| `onAutoClose` .    | `(event: Event) => void`                     |               | no       | Callback on close on click outside of Dropdown                  |
-| `isFullWidth`      | `boolean`                                    | `false`       | no       | Whether is component displayed in full width                    |
-| `placement`        | [`DropdownPlacement`][dropdownplacement]     | `bottom-left` | no       | Alignment of the component                                      |
-| `breakpoint`       | [`DropdownBreakpoint`][dropdownbreakpoint]   | -             | no       | Breakpoint to switch from the full-width to the auto-width mode |
-| `UNSAFE_className` | `string`                                     | -             | no       | Wrapper custom classname                                        |
-| `UNSAFE_style`     | `CSSProperties`                              | -             | no       | Wrapper custom style                                            |
+| Prop name          | Type                                             | Default       | Required | Description                                                                                                                        |
+| ------------------ | ------------------------------------------------ | ------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `id`               | `string`                                         | `<random>`    | no       | Component id                                                                                                                       |
+| `renderTrigger`    | `(render: DropdownRenderProps) => ReactNode`     | -             | no       | Properties for trigger render                                                                                                      |
+| `enableAutoClose`  | `boolean`                                        | `true`        | no       | Enables close on click outside of Dropdown                                                                                         |
+| `onAutoClose` .    | `(event: Event) => void`                         |               | no       | Callback on close on click outside of Dropdown                                                                                     |
+| `isFullWidth`      | `boolean`                                        | `false`       | no       | Whether is component displayed in full width                                                                                       |
```

Comment:
@adamkudrna `isFullWidth` is not compatible with the `fullWidthMode` prop. Eg. `isFullWidth` is the same as `fullWidthMode="all"`, so I think that we should also deprecate this prop or make and alias from them to `fullWidhtMode="all"`. What do you think?

Reply:
I would deprecate it as well.

Reply:
@crishpeen Thanks, deprecated.